anvio/__init__.py

```diff
@@ -3669,7 +3669,27 @@ def TABULATE(table, header, numalign="right", max_width=0):
                      "a comma-separated list. The default stats are 'detection' and "
                      "'mean_coverage_Q2Q3'. To see a list of available stats, use this flag "
                      "and provide an absolutely ridiculous string after it (we suggest 'cattywampus', but you do you)."}
-    )
+    ),
+    'mode': (
```

Comment:
Please change the new community parameter to `pan-mode` so we don't run into trouble in the long run.